### ISSUE TITLE
[Question] Use Amazon list as a way for the code to check stock of GPU's...

### ISSUE THREAD

**Issue by shootlikekd** [TreborNamor/Agressive-Bestbuy-Bot#18]
Hello and thanks for the bot it has worked great!

After playing around with the bot I noticed that it only worked on store pages, which some cards don't exist. I was wondering if there was a way to make the bot scan a amazon list page (Ex: https://www.amazon.com/hz/wishlist/dl/invite/2yt81k3?ref_=wl_share). This would allow a user to list the items they want to be botted and also add in specific cards that are not on store pages.

So I tried messing around with the code myself but I have little knowledge in HTML and Python. I'm guessing the problem lies in how the "add to cart" button differs between the two pages?

If you could offer me any input I would be greatly appreciative.

**Comment by momobadarni** [comment]
Hey have you had any successful purchases with your list setup?

**Comment by shootlikekd** [comment]
> Hey have you had any successful purchases with your list setup?

Not really, with the way amazon updates their pages most of the time a scalper will own the main add to cart button. Everyone else's bots will get the card before a legit "Add to Cart" button shows as the main option. So the only way to beat other bots is by scanning the offer listings page for each card individually, setting a parameter for pricing (min:400, max:500), and refreshing until an option in that price range becomes available.
